sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -320,10 +320,10 @@ class GetManifestResult:
     :ivar str digest: The manifest's digest, calculated by the registry.
     """
 
-    def __init__(self, **kwargs):
-        self.manifest = kwargs.get("manifest")
-        self.media_type = kwargs.get("media_type")
-        self.digest = kwargs.get("digest")
+    def __init__(self, *, manifest: Mapping[str, Any], media_type: str, digest: str) -> None:
```

Comment:
Will a user ever need to create this type or is output-only? We're adding required keyword-only arguments when previously there were none.

Reply:
It's used in output only. 
@annatisch do you think we can update the apiview to avoid mypy errors by casting each of them?

Reply:
Was it ever possible that any of these values from the service could be None?
I think I would probably prefer to keep the constructor unbroken, but I also don't want to type these attributes as Optional if they aren't.

Reply:
This model is public though used in output only. I haven't seen `None` in service response so far, but other languages like .Net set each property default value to `None`.
Here I changed the constructor back.